doc/attr.xml

```diff
@@ -1241,6 +1241,32 @@ gap> DigraphLoops(gr);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DigraphMycielskian">
+<ManSection>
+  <Attr Name="DigraphMycielskian" Arg="digraph"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>digraph</A> is a symmetric digraph, then <C>DigraphMycielskian</C> returns the Mycielskian of <A>digraph</A>. If <A>digraph</A> is not symmetric, then an error message is returned.<P/>
```

Comment:
Could you please delete the second sentence here? We (seem to) have adopted the convention in this package that we don't include such sentences, describing than an error is given when the argument doesn't match the requirements.